src/eXpansion/Framework/Notifications/Services/Notifications.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: php_r
+ * Date: 25.2.2018
+ * Time: 21.50
+ */
+
+namespace eXpansion\Framework\Notifications\Services;
+
+
+use eXpansion\Framework\Core\Model\UserGroups\Group;
+use eXpansion\Framework\Notifications\Plugins\Gui\NotificationUpdater;
+use Psr\Log\LoggerInterface;
+
+class Notifications
+{
+    /**
+     * @var NotificationUpdater
+     */
+    private $notificationsUpdater;
+    /** @var Group */
+    private $group;
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * Notifications constructor.
+     * @param Group               $group
+     * @param NotificationUpdater $notificationUpdater
+     * @param LoggerInterface     $logger
+     */
+    public function __construct(
+        $group,
+        NotificationUpdater $notificationUpdater,
+        LoggerInterface $logger
+    ) {
+        $this->notificationsUpdater = $notificationUpdater;
+        $this->group = $group;
+        $this->logger = $logger;
+    }
+
+    /**
+     * Sends notification as info level
+     * Notifications are automatically logged
+     *
+     * @param string     $title Title of the notification
+     * @param string     $message Message of notification
+     * @param int        $duration use 0 for permanent
+     * @param Group|null $group
+     */
+    public function info($message, $title = "Info", $duration = 5500, $group = null)
+    {
+        $this->logger->info("[Notification] ".$message, ["title" => $title]);
```

Comment:
Hmm the logger ...

Thse are notifications no real application errors or warnings. You look into the logs to see why the application has crashed.  If you want to log it should be a different channel at least? 

What do you think ?

Reply:
I removed the logger then.